src/cli/cli.cpp

```diff
@@ -554,6 +595,7 @@ void Interpreter::ProcessCounters(int argc, char *argv[])
             sServer->OutputFormat("    RxErrSec: %d\r\n", counters->mRxErrSec);
             sServer->OutputFormat("    RxErrFcs: %d\r\n", counters->mRxErrFcs);
             sServer->OutputFormat("    RxErrOther: %d\r\n", counters->mRxErrOther);
+            otFreeMemory(counters);
```

Comment:
@nibanks , can we use smart-ptr/shared-ptr on windows (so the returned items are freed automatically when they go out of scope) so to  avoid the need for `otFreeMemory()` calls?